cypress/integration/all/setup/setup.spec.ts

```diff
@@ -17,33 +45,20 @@ describe("Setup", () => {
     cy.url().should("match", /\/setup$/);
     cy.contains(/setup/i);
 
-    // Page 1
-    cy.findByPlaceholderText(/full name/i).type("Test name");
-
-    cy.findByPlaceholderText(/email/i).type("test@example.com");
+    fillOutForm();
 
-    cy.findByPlaceholderText(/^password/i)
-      .first()
-      .type(GOOD_PASSWORD);
-
-    cy.findByPlaceholderText(/confirm password/i)
-      .last()
-      .type(GOOD_PASSWORD);
-
-    cy.contains("button:enabled", /next/i).click();
-
-    // Page 2
-    cy.findByPlaceholderText(/organization name/i).type("Fleet Test");
-
-    cy.contains("button:enabled", /next/i).click();
+    cy.url().should("match", /\/hosts\/manage$/i);
+    cy.contains(/all hosts/i);
+  });
 
-    // Page 3
-    cy.contains("button:enabled", /next/i).click();
+  it("shows messaging when there is an error during setup", () => {
+    cy.visit("/");
+    cy.intercept("POST", "/api/v1/setup", { forceNetworkError: true });
 
-    // Page 4
-    cy.contains("button:enabled", /confirm/i).click();
+    fillOutForm();
 
-    cy.url().should("match", /\/hosts\/manage$/i);
-    cy.contains(/all hosts/i);
+    cy.findByText(
+      "We were unable to configure Fleet. If your Fleet server is behind a proxy, please ensure the server can be reached."
```

Comment:
Great conversation for our next interface weekly. Do we want to test network errors elsewhere in the app? @lukeheath @ghernandez345 @gillespi314

Reply:
yes for sure. I have thought more about testing errors states and feel they will be split between our e2e and integration tests. But I think at a minimum we really do need to test the most common errors a user can encounter in our e2e layer.

That being said the architecture of our code will definitely have an influence on exactly what level we are testing things, which is what we can cover more in our meeting.